amundsen_application/static/js/components/TableDetail/RequestMetadataForm/tests/index.spec.tsx

```diff
@@ -65,6 +65,59 @@ describe('RequestMetadataForm', () => {
   
   // TODO
   describe('render', () => {
+    let props;
+    let wrapper;
+    let element;
+    beforeAll(() => {
+      const setupResult = setup();
+      props = setupResult.props;
+      wrapper = setupResult.wrapper;
+    });
+  
+    it('renders from label', () => {
+      element = wrapper.find('#sender-form-group');
+      expect(element.find('label').text()).toEqual('From');
```

Comment:
For all these text values you can import the constants and use them instead. That way if we decide to change the text the tests don't have to be touched.